### PR TITLE
Upgrade to Play 2.8.8

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Upgrade to Play 2.8.8

### CODE REVIEW

File: common/app/common/AutoRefresh.scala [hunk 1]
```diff
@@ -24,18 +24,16 @@ abstract class AutoRefresh[A](initialDelay: FiniteDuration, interval: FiniteDura
       a <- get
     } yield Future.successful(a)).getOrElse(refresh())
 
+  class Task(implicit executionContext: ExecutionContext) extends Runnable {
+    def run() {
+      refresh().map { data => agent.send(Some(data)) }
```
Comment (jfsoul) [inline common/app/common/AutoRefresh.scala:29]: Would be nice to keep the same behaviour as before with the logging?

File: sport/app/cricket/feed/CricketThrottler.scala [hunk 2]
```diff
@@ -20,10 +19,18 @@ class CricketThrottlerActor()(implicit materializer: Materializer) extends Actor
...
   private case class TaskWithSender[+T](sender: ActorRef, task: () => Future[T])
 
+  val completionMatcher: PartialFunction[Any, CompletionStrategy] = { case e => CompletionStrategy.immediately }
+  val failureMatcher: PartialFunction[Any, Throwable] = { case e => new Throwable() }
+
   val throttler: ActorRef = Source
-    .actorRef[CricketThrottledTask[Nothing]](bufferSize = 1024, OverflowStrategy.dropNew)
+    .actorRef[CricketThrottledTask[Nothing]](
+      completionMatcher = completionMatcher,
+      failureMatcher = failureMatcher,
```
Comment (jfsoul) [inline sport/app/cricket/feed/CricketThrottler.scala:28]: Forgive my laziness for not googling, what do these do?

**Review by jfsoul — APPROVED** [review]
Seems reasonable, couple of questions  👍

File: common/app/model/DatesAndTimes.scala [hunk 3]
```diff
@@ -97,9 +97,9 @@ object GUDateTimeFormatNew {
   }
   def formatDateTimeForDisplayGivenEdition(date: DateTime, edition: Edition): String = {
     val timezone = edition.timezone
-    date.toString(DateTimeFormat.forPattern("E d MMM yyyy HH.mm").withZone(timezone)) + " " + timezone.getShortName(
-      date.getMillis,
-    )
+    val timeZoneString1 = timezone.getShortName(date.getMillis)
+    val timeZoneString2 = if (timeZoneString1 == "GMT-04:00") "EDT" else timeZoneString1
+    date.toString(DateTimeFormat.forPattern("E d MMM yyyy HH.mm").withZone(timezone)) + " " + timeZoneString2
```
Comment (nicl) [inline common/app/model/DatesAndTimes.scala:102]: What is the motivation for these changes? I wasn't sure how this differs from the previous code and why. Improving naming here to something more descriptive for `timeZoneString1/2` might help to.

Comment (shtukas) [reply common/app/model/DatesAndTimes.scala:102]: Oh, yeah. I was about to describe that change in the PR description. 

For some reasons `timezone.getShortName(date.getMillis)` no longer returns "EDT" as it used to do, but now returns "GMT-04:00". So I added a line of code. 

I get your point about the naming, let me update that :)

File: common/app/common/AutoRefresh.scala (hunk 1 above)
Comment (shtukas) [reply common/app/common/AutoRefresh.scala]: Done. Cheers

File: sport/app/cricket/feed/CricketThrottler.scala (hunk 2 above)
Comment (shtukas) [reply sport/app/cricket/feed/CricketThrottler.scala:28]: I am assuming you were asking about the agent itself. It retrieves cricket  data from PA :)

Comment (shtukas) [reply sport/app/cricket/feed/CricketThrottler.scala:28]: Otherwise we are setting run parameters for the agent
```
  val completionMatcher: PartialFunction[Any, CompletionStrategy] = { case e => CompletionStrategy.immediately }
  val failureMatcher: PartialFunction[Any, Throwable] = { case e => new Throwable() }
```
